app/models/propertyrepresentation/ManageMultiplePropertiesOptions.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.propertyrepresentation
+
+import models.{EnumFormat, NamedEnum, NamedEnumSupport}
+
+sealed trait ManageMultiplePropertiesOptions extends NamedEnum {
```

Comment:
Is this our preferred way of doing enums?

Reply:
that is the pattern that I am aware of. @DanielConnelly could confirm

Reply:
this is not a great standard for enum pattern if we use base scala Enumeration. if we want to serialise these as objects then we use sealed traits.